src/cephadm/cephadm

```diff
@@ -79,6 +79,7 @@ else:
     from urllib2 import urlopen, HTTPError
 
 container_path = ''
+cached_stdin = None
```

Comment:
We're in desperate need for a 

```python
class Cephadm(object, args):
    def __init__(self):
        self.cached_stdin = None
        self.container_path = ''
        self.args = args

    def command_...(self): ...
```

to properly separate the application logic from utilities.

I see the need for making progress here, but adding these kind of global variables will make it even more involved to fix this in the future.